navigator.lua

```diff
@@ -4,9 +4,19 @@
 -- Contributors: SteveJobzniak
 -- URL: https://github.com/donmaiq/mpv-filenavigator
 --
+
+local utils = require 'mp.utils'
+
+ON_WINDOWS = (package.config:sub(1,1) ~= '/')
+WINDOWS_ROOTDIR = false
+WINDOWS_ROOT_DESC = "Select drive"
+SEPARATOR_WINDOWS = "\\"
+
+SEPARATOR = "/"
+
 local settings = {
   --fallback if no file is open, should be a string that points to a path in your system
-  defaultpath = os.getenv("HOME") or "/",
+  defaultpath = utils.join_path(os.getenv("USERPROFILE"), "Desktop"):gsub(SEPARATOR, SEPARATOR_WINDOWS)..SEPARATOR_WINDOWS or os.getenv("HOME") or "/",
```

Comment:
This breaks on linux:

`[navigator] Lua error: /home/ekayjon/.config/mpv/scripts/navigator.lua:20: bad argument #1 to 'join_path' (string expected, got nil)`

```suggestion
  defaultpath = ON_WINDOWS and utils.join_path(os.getenv("USERPROFILE"), "Desktop"):gsub(SEPARATOR, SEPARATOR_WINDOWS)..SEPARATOR_WINDOWS or os.getenv("HOME") or "/",
```